src/Package/Car.java

```diff
@@ -1,15 +1,14 @@
 package Package;
 
-public class Car {
-
-    public String modelName;
-    public int wheelsCount;
-
-    public void updateTyre() {
-        System.out.println("Меняем покрышку");
+public class Car extends Vehicle{
+    public Car(String modelName, int wheelsCount) {
+        super(modelName, wheelsCount);
     }
-
     public void checkEngine() {
         System.out.println("Проверяем двигатель");
```

Comment:
```suggestion
        System.out.println("Проверяем двигатель авто");
```
чтобы подчеркнуть, что реализации разные и есть полиморфизм